### PR TITLE
add Blocks [ver1.1.3]

### PR DESCRIPTION
**Pull request by shoki-3738**

### CODE REVIEW

File: src/pocketmine/block/BlockIds.php [hunk 1]
```diff
@@ -304,6 +304,11 @@
 	const FROSTED_ICE = 207;
 	const END_ROD = 208;
 
+	const MAGMA_BLOCK = 213;
+	const NETHER_WART_BLOCK = 214;
+	const RED_ NETHER_BRICK = 215;
```
Comment (Leo3418) [inline src/pocketmine/block/BlockIds.php]: Syntax error

**Comment by shoki-3738** [comment]
Sorry

File: src/pocketmine/block/BlockIds.php (hunk 1 above)
Comment (shoki-3738) [inline src/pocketmine/block/BlockIds.php]: sorry, fix

**Comment by andris155** [comment]
NETHER_WART_BLOCK already used.

BlockIds.php
const NETHER_WART_BLOCK = 115;
Block.php
self::$list[self::NETHER_WART_BLOCK] = NetherWart::class;

**Comment by shoki-3738** [comment]
Changed to NETHER_WART_BLOCK_

**Pull request closed without merging**